src/constraints/storage_constraints.jl

```diff
@@ -99,6 +107,32 @@ function add_elec_storage_dispatch_constraints(m, p, b; _n="")
             sum(m[Symbol("dvStorageEnergy"*_n)][b])
         )
     end
+
+    if p.s.storage.attr[b].is_ldes
```

Comment:
I'd suggest that we make the nomenclature more generalized because users could want to consider non LDES storage techs with fixed duration. We could have a bool input called duration_is_fixed, but actually I don't think we really need this to be an input at all. We can just have the duration input default to nothing and a non-exposed struct field duration_is_fixed that is set to true if the user provides a value for duration. I can make this change if it sounds alright with y'all.

Reply:
done